keyboards/duck/orion/v2_5/rules.mk

```
@@ -0,0 +1,38 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
+BACKLIGHT_DRIVER = custom
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+RGBLIGHT_ENABLE = yes
+
+CUSTOM_MATRIX = yes
```

Please convert this to matrix lite
```suggestion
CUSTOM_MATRIX = lite
```

https://docs.qmk.fm/#/custom_matrix

---

Tried this change, but it gave errors when compiling. Linking failed with duplicate definitions.
changes this back to yes, but comitted all other suggested changes. compiles fine, tested and verified to be working.

---

Just this change on its own is not enough, you need to clean up the matrix.c a little:
https://docs.qmk.fm/#/custom_matrix?id=lite

---

Sorry for not being more clear.  Yeah, you'd want to remove the kb/user callbacks, and you'd want to change the matrix_init and matrix_scan functions to match what the documentation shows. 

You can also remove the matrix_print function, as well.  And the debouncing code. As that's handled automatically by the matrix lite config.